lib/msal-browser/src/index.ts

```diff
@@ -1,6 +1,7 @@
 // App and config
 export { PublicClientApplication } from "./app/PublicClientApplication";
 export { Configuration } from "./config/Configuration";
+export { InteractionType } from "./utils/BrowserConstants";
```

Comment:
We may want to change the casing of the values to capital-case  ("Popup" instead of "POPUP") now that this being exported.

Reply:
Do you mean for the enum keys? The values are all lowercase "popup"

Reply:
Done